native/packages/react-native-bpk-component-badge/src/BpkBadge.js

```diff
@@ -97,6 +100,9 @@ const styles = StyleSheet.create({
     borderBottomLeftRadius: borderRadiusSm,
     borderBottomRightRadius: 0,
   },
+  accessoryView: {
+    marginRight: spacingSm,
```

Comment:
`marginEnd` is better - works with RTL then.

Reply:
Strangely it seemed to work correctly with `marginRight`, but I'll use RTL friendly ones just to be sure

<img width="545" alt="screenshot 2018-11-22 at 17 37 29" src="https://user-images.githubusercontent.com/30267516/48917174-4faf2080-ee7d-11e8-9376-c57c3e6cd704.png">